<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesPalette_TableBase</class>
 <widget class="QWidget" name="PropertiesPalette_TableBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>398</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Styles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelTable">
        <property name="text">
         <string>Table:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="TableStyleComboBox" name="tableStyleCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Table style of currently selected table</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="buttonClearTableStyle">
        <property name="toolTip">
         <string>Remove Direct Table Formatting</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCells">
        <property name="text">
         <string>Cells:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CellStyleComboBox" name="cellStyleCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Cell style of currently selected cell</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="buttonClearCellStyle">
        <property name="toolTip">
         <string>Remove Direct Cell Formatting</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fillLayout">
     <property name="title">
      <string>Fill</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="fillLayout1" stretch="0,0">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <layout class="QFormLayout" name="fillLayout1_1">
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <property name="verticalSpacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="fillColorLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Color:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ColorCombo" name="fillColor">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="fillLayout1_2">
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <property name="verticalSpacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="fillShadeLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Shade:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ScrSpinBox" name="fillShade">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string> %</string>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="bordersLayout">
     <property name="title">
      <string>Borders</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="bordesLayout1">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="bordersLayout1_1">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QToolButton" name="addBorderLineButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="removeBorderLineButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="buttonSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="borderLineList">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TableSideSelector" name="sideSelector">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="bordersLayout2" stretch="0,0">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <layout class="QFormLayout" name="bordersLayout2_1">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <property name="verticalSpacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="borderLineWidthLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Width:</string>
            </property>
            <property name="buddy">
             <cstring>borderLineWidth</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="borderLineColorLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Color:</string>
            </property>
            <property name="buddy">
             <cstring>borderLineColor</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ColorCombo" name="borderLineColor">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ScrSpinBox" name="borderLineWidth">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="bordersLayout2_2">
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <property name="verticalSpacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="borderLineStyleLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Type:</string>
            </property>
            <property name="buddy">
             <cstring>borderLineStyle</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="LineCombo" name="borderLineStyle">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="borderLineShadeLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Shade:</string>
            </property>
            <property name="buddy">
             <cstring>borderLineShade</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ScrSpinBox" name="borderLineShade">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string> %</string>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LineCombo</class>
   <extends>QComboBox</extends>
   <header>ui/linecombo.h</header>
  </customwidget>
  <customwidget>
   <class>TableSideSelector</class>
   <extends>QFrame</extends>
   <header>ui/tablesideselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorCombo</class>
   <extends>QComboBox</extends>
   <header>ui/colorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>TableStyleComboBox</class>
   <extends>QComboBox</extends>
   <header>ui/stylecombos.h</header>
  </customwidget>
  <customwidget>
   <class>CellStyleComboBox</class>
   <extends>QComboBox</extends>
   <header>ui/stylecombos.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
